modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -2325,6 +2390,124 @@ definition {
 		ObjectAdmin.assertNoResultsWereFound();
 	}
 
+	@description = "LPS-169994 - When the action is disabled it must not be executed even if the user has the proper permission"
+	@priority = 5
+	test CanNotSeeDesativedStandaloneActionWithPermission {
+		property portal.acceptance = "true";
+
+		task ("Given: An object with a standalone action turned off") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 185",
+				objectName = "CustomObject185",
+				pluralLabelName = "Custom Objects 185");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "true",
+				objectName = "CustomObject185");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject185");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 185");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Action Label",
+				actionName = "ActionName",
+				active = "true",
+				errorMessage = "Error message test",
+				inputMethod = "true",
+				inputValue = "New Entry Test",
+				objectLabel = "Custom Object 185",
+				thenAction = "Add an Object Entry",
+				whenAction = "Standalone");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject185",
+				value = "Entry Test");
+		}
+
+		task ("When: The user has permissions to trigger a standalone action") {
+			JSONRole.addRegularRole(roleTitle = "Regular Role");
```

Comment:
Since the Admin user has all the permissions, and we want to assert that if the action is disabled, it can't the trigged, then it's no necessary to add another Role, we can use the Test Test user for this case.
The test scenario was a little tricky here.

In my view we can remove this step, and let a single When.